evaluation/logic_reasoning/instruction.txt

```diff
@@ -1,6 +1,6 @@
-You are a helpful assistant assigned with logic reasoning task. You need to determine the correctness of a query given some facts and fules. 
+You are a helpful assistant assigned with logic reasoning task. You need to determine the correctness of a query given some facts and fules.
```

Comment:
It's fine to fix this given that we haven't run benchmarks for logic reasoning systematically yet.

```suggestion
You are a helpful assistant assigned with logic reasoning task. You need to determine the correctness of a query given some facts and rules.
```